src/GHCMod.hs

```diff
@@ -107,7 +110,6 @@ getFileSourceFromStdin = do
         then fmap (x:) readStdin'
         else return []
 
--- Someone please already rewrite the cmdline parsing code *weep* :'(
```

Comment:
Yes, that's been there for a while now. I thought that this is as good a time as any to remove it.